sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Version.java

```diff
@@ -176,6 +186,14 @@ public Version removeQualifier() {
    * Two versions are compatible when they are identical except for qualifier.
    */
   public boolean isCompatibleWith(Version version) {
-    return this.removeQualifier().equals(version.removeQualifier());
+    return this.removeQualifier().compareTo(version.removeQualifier()) == 0;
+  }
+
+  private static int compareTo(String normalized, String otherNormalized) {
```

Comment:
This method is badly named, since collision with `Comparable::compareTo`. `compareToNormalizedParts` might be better.